CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## 0.15.0 - 2021-07-<???>
+
+### Added
+
+- Change reload to be configurable with glob patterns (#820) 08/08/21
```

Comment:
we need to add a comment from : https://github.com/encode/uvicorn/pull/820#discussion_r506730032